qcodes/instrument/meta/meta_instrument.py

```diff
@@ -0,0 +1,185 @@
+from typing import List, Dict, Union, Any, Optional
+
+from functools import partial
+
+from qcodes.instrument.meta.meta_parameter import MetaGroup, MetaParameter
+from qcodes.instrument.parameter import Parameter
+from qcodes.instrument.base import InstrumentBase
+from qcodes.station import Station
+
+import logging
+
+_log = logging.getLogger(__name__)
+
+
+class MetaInstrument(InstrumentBase):
+    """MetaInstrument class for creating a meta instrument that aliases one or more 
+    parameter endpoints from real instruments.
+
+    Example usage in instrument YAML:
```

Comment:
This docstring could do with a bit more detail. It is hard to see from the current docstring how this is meant to be called